cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-error-after-typing-non-existing-item-location.cy.js

```diff
@@ -0,0 +1,73 @@
+import Permissions from '../../../support/dictionary/permissions';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import FileManager from '../../../support/utils/fileManager';
+import getRandomPostfix from '../../../support/utils/stringTools';
+
+let user;
+const item = {
+  barcode: getRandomPostfix(),
+  instanceName: `instance-${getRandomPostfix()}`,
+};
+const itemBarcodesFileName = `itemBarcodes_${getRandomPostfix()}.csv`;
+
+describe('Bulk Edits', () => {
+  describe('Bulk Edit - Items', () => {
```

Comment:
please change to 
`describe('bulk-edit', () => {
  describe('in-app approach', () => {`
so the tests are sorted correctly in the allure

Reply:
fixed